src/ctia/domain/entities.clj

```diff
@@ -118,7 +118,11 @@
     (long-id->entity-type id-str)))
 
 (defn un-store [record]
-  (dissoc record :created :modified))
+  #_
+  (when (= "incident" (:type record))
+    (prn "un-store incident" record))
+  (cond-> (dissoc record :created :modified)
+    (= "incident" (:type record)) (dissoc :intervals)))
```

Comment:
this part is generic, specific behaviour should be handled in the incident store.
Example for sighting: https://github.com/threatgrid/ctia/blob/master/src/ctia/entity/sighting/es_store.clj